src/test/java/seedu/address/testutil/IncomeListBuilder.java

```diff
@@ -0,0 +1,34 @@
+package seedu.address.testutil;
+
+import seedu.address.model.IncomeList;
+import seedu.address.model.income.Income;
+
+/**
+ * A utility class to help with building IncomeList objects.
+ * Example usage: <br>
+ *     {@code AddressBook ab = new AddressBookBuilder().withPerson("John", "Doe").build();}
+ */
+public class IncomeListBuilder {
```

Comment:
Do we really need an `IncomeListBuilder`? 🤔 Feels abit like overengineering in this case since we're only just adding to the list